### PR TITLE
Add write a proposal in the issue before submitting a PR

### PR DESCRIPTION
**Pull request by iwiznia**
### Details
Clarifying about sending a proposal.

Just a readme change, no tests.

### CODE REVIEW

**Comment by iwiznia** [comment]
I have read the CLA Document and I hereby sign the CLA

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
 
 ## Submitting a Pull Request
 #### Proposing a Change
+1. *Before writing any code* write a proposal for the solution in the issue
```
Comment (Jag96) [inline CONTRIBUTING.md:27]: Can you add more detail about what a proposal entails? Maybe something like

```
**Before writing any code**, please post a proposal of the solution you plan to implement as a comment in the GH issue. This should include a brief technical explanation of the changes you will make.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,6 +24,8 @@ We are currently managing payment via Upwork. If you'd like to be paid for your
...
+1. Wait for Expensify to review and provide feedback on the proposal within the GH issue, and assign the GH issue to you
```
Comment (Jag96) [inline CONTRIBUTING.md:28]: I'd also mention here `Please do not move forward with creating a Pull Request until your proposal is approved`

**Review by Jag96 — CHANGES_REQUESTED** [review]
Just some notes for more details here

**Comment by iwiznia** [comment]
Updated

**Review by Jag96 — APPROVED** [review]

**Pull request merged**